build-tools/packages/build-tools/src/common/fluidRepo.ts

```diff
@@ -55,6 +55,11 @@ export interface IFluidBuildConfig {
 	branchReleaseTypes?: {
 		[name: string]: VersionBumpType | PreviousVersionStyle;
 	};
+
+	/**
+	 * Configuration for the `generate:releaseNotes` command.
```

Comment:
Presumably default settings are used if this isn't specified (undefined)?

Reply:
No, actually the command will just error if this isn't provided. I'll clarify in the comment.